Not a real commit (#1)
jadekler/aip: aip/0194.md

unintended state changes. Most CRUD RPCs which take a unique identifier are

> considered retryable (see more below in "Idempotency and pragmaticism").

Author: Should this me `pragmatism`? I found only an [obscure branch of philosophy](https://en.wikipedia.org/wiki/Pragmaticism) for `pragmaticism`.

> **Transactional** requests are requests which are part of a transactions. These

Author: ```suggestion
**Transactional** requests are requests which are part of a transaction. These
```

Streaming requests can be categorized as **server streaming**,

> **client streaming**, and **bidirectional streaming**.

Author: This feels like it should be either
```suggestion
**client streaming**, or **bidirectionally streaming**.
```
or
```suggestion
Streaming requests can use **server streaming**,
**client streaming**, or **bidirectional streaming**.
```

**Bidirectional streaming** request retryability is highly dependent on the type
of data being streamed.

## Codes that should never be retried

- `OK`: The request succeeded.

> - `CANCELLED`: A user can cancel a request, which we should honour.

Author: General comment for the document; Is using the first person correct here? Alternatively
```suggestion
- `CANCELLED`: A user can cancel a request, which should be honoured.
```

**Bidirectional streaming** request retryability is highly dependent on the type

> of data being streamed.

Author: Maybe
```suggestion
of data being exchanged.
```
To differentiate it from the previous line.

> The definition of idempotency is unfortunately vague in this regard. Idempotent

Author: What definition? Reference a source.

However, it is easy to see that _one of_ these requests would have to receive

> `ALREADY_EXISTS`, even if no retrying happening.

Author: ```suggestion
 `ALREADY_EXISTS`, even if no retrying is happening.
```
